spec/spec_helper.rb

```diff
@@ -45,5 +46,11 @@ module BlockScore
 
     config.order = :random
     Kernel.srand config.seed
+
+    config.around do |example|
+      Timeout.timeout(10) do
```

Comment:
The `example#run` method is also aliased to `#call`, so you could change this to:

```ruby
Timeout.timeout(10, &example)
```